html/comparison.html.haml

```diff
@@ -495,7 +495,7 @@
                 %span.summary-part{"ng-show" => "diff.changed_managed_files.only_in2"}
                   {{diff.meta.description_b}}: {{diff.changed_managed_files.only_in2.files.length || 0}} files
                 %span.summary-part{"ng-show" => "diff.changed_managed_files.common"}
-                  %a{:class=>"show-common-elements", href: "#changed_managed_files_both"}
+                  %a{:class=>"show-common-elements", href: "#changed_managed_files_both"}<
```

Comment:
Line is too long. [90/80]<br>Avoid defining `class` in attributes hash for static class names<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace